app/assets/stylesheets/views/_history.scss

```diff
@@ -0,0 +1,19 @@
+.home-history {
+  .image {
```

Comment:
Same comment about using the figure component if possible.

Reply:
I don't think it will be possible, see the images on https://www.gov.uk/government/history

Reply:
Yeah, good point, I hadn't realised that top border and caption layout was part of the component. I'm wondering if we should wrap the main page content for https://government-f-msw-10-dow-u2ibe2.herokuapp.com/government/history/10-downing-street in the govspeak component and let it handle the rendering. 

It looks like govspeak already handles images, e.g: https://www.gov.uk/government/case-studies/the-women-who-are-making-their-countries-wealthier we just need to make sure it's marked up correctly.

Reply:
I'll give it a shot

Reply:
It looks pretty good